examples/index.json

```
@@ -1035,6 +1015,32 @@
             }
         ]
     },
+    {
+        "title": "Access Control",
+        "column": 3,
+        "category": "Working over the network",
+        "samples": [{
+                "name": "Secured Service With JWT",
+                "url": "secured-service-with-jwt"
+            },
+            {
+                "name": "Secured Service With Basic Auth",
```

```suggestion
                "name": "Secured Service with Basic Auth",
```

Normally, we don't capitalize conjunctions.